src/main/java/org/jitsi/jicofo/recording/jibri/JibriSession.java

```diff
@@ -781,4 +796,23 @@ void onSessionStateChanged(
                 JibriIq.Status    newStatus,
                 JibriIq.FailureReason         failureReason);
     }
+
+    public enum ErrorReason {
```

Comment:
What about calling this something like `JibriResult` and including a 'Success' value in it that we can use instead of returning null?

Reply:
okay

Reply:
I don't know if it feels better or worse. What I don't like is that now Result mixes success with errors, but on the other hand it reads better when checks for Result.SUCCESS instead of null.

Reply:
maybe Exception would be best